shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Rules/TestPoolreap.hs

```diff
@@ -44,17 +46,21 @@ import Shelley.Spec.Ledger.STS.PoolReap
     prUTxOSt,
     pattern PoolreapState,
   )
+import Shelley.Spec.Ledger.Slot (EpochNo (..))
 import Shelley.Spec.Ledger.UTxO (balance)
 import Test.QuickCheck (Property, conjoin)
-import Test.Shelley.Spec.Ledger.Rules.TestPool (getRetiring)
+
+getRetiring :: PState era -> Map (KeyHash 'StakePool era) EpochNo
+getRetiring = _retiring
```

Comment:
Particular reason for this renaming?

Reply:
Especially since it's not been added to the export list...

Reply:
Oh, I see, it used to be elsewhere. Maybe take the opportunity to just replace with `_retiring`?

Reply:
ah yes, removed (curiously, still needed that annotation before removing fixation to crypto C)